samples/msal-node-samples/on-behalf-of-distributed-cache/README.md

```diff
@@ -0,0 +1,90 @@
+   - Type a key description (for instance `app secret`),
+   - Select one of the available key durations (**In 1 year**, **In 2 years**, or **Never Expires**) as per your security posture.
+   - The generated key value will be displayed when you select the **Add** button. Copy the generated value for use in the steps later.
+    > :warning: In production, use certificates with Azure Key Vault instead of secrets. See [certificate-credentials.md](../../../lib/msal-node/docs/certificate-credentials.md) and [key-vault.md](../../../lib/msal-node/docs/key-vault-managed-identity.md) for more information and examples.
+1. In the app's registration screen, select the **Expose an API** blade to the left to open the page where you can declare the parameters to expose this app as an API for which client applications can obtain [access tokens](https://docs.microsoft.com/azure/active-directory/develop/access-tokens) for.
+The first thing that we need to do is to declare the unique [resource](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-auth-code-flow) URI that the clients will be using to obtain access tokens for this API. To declare an resource URI, follow the following steps:
+   - Select `Set` next to the **Application ID URI** to generate a URI that is unique for this app.
+   - For this sample, accept the proposed Application ID URI (`api://{clientId}`) by selecting **Save**.
+1. All APIs have to publish a minimum of one [scope](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-auth-code-flow#request-an-authorization-code) for the client's to obtain an access token successfully. To publish a scope, follow the following steps:
+   - Select **Add a scope** button open the **Add a scope** screen and Enter the values as indicated below:
+        - For **Scope name**, use `access_as_user`.
+        - Select **Admins and users** options for **Who can consent?**.
+        - For **Admin consent display name** type `Access msal-node-webapi`.
+        - For **Admin consent description** type `Allows the app to access msal-node-webapi as the signed-in user.`
+        - For **User consent display name** type `Access msal-node-webapi`.
+        - For **User consent description** type `Allow the application to access msal-node-webapi on your behalf.`
+        - Keep **State** as **Enabled**.
+        - Select the **Add scope** button on the bottom to save this scope.
+1. On the left hand side menu, select the `Manifest` blade.
+   - Set `accessTokenAcceptedVersion` property to **2**.
+   - Modify the `"knownClientApplications": []` array to contain the client ID/app ID of the client application(S) that will call this web API (for example `knownClientApplications": ["APP_ID_OF_THE_CLIENT_APP"]`)
+   - Click on **Save**.
+
+Before running the sample, you will need to replace the values in the configuration object (see [app.ts](./src/app.ts)):
+
+```typescript
+const appConfig: AppConfig = {
```

Comment:
Consider asking the app developers to also configure Redis.

Reply:
done!